src/components/bookRenderer.js

```diff
@@ -13,7 +13,7 @@ const BookRenderer = () => {
 
   return (
     <>
-      <div>
+      <div className="books-container">
```

Comment:
- [ Required ] Great job, adding the UI styling to the project, however, I think you assumed the drawing canvas was also part of the project design and therefore you introduced extra `paddings` and `margins` on your design. Please see the screenshot below detailing the extra white space `( Arrows in red )` I'm talking about and rectify this.

- [ Required ] The margin between any two books on the list is also wider than on the `Zeplin` design. Kindly rectify this.

## Zeplin design
![Screenshot from 2023-02-11 05-16-52](https://user-images.githubusercontent.com/16392046/218234315-26da9a5c-ab31-492a-9357-db3b0cbdbb11.png)

## Your design
![Screenshot from 2023-02-11 05-18-10](https://user-images.githubusercontent.com/16392046/218234671-8d56a7d0-746c-4e10-8736-cd0fd6ba6c16.png)

Reply:
done